recipes/mne-python/construct.yaml

```
@@ -88,6 +88,7 @@ specs:
   - mne-lsl =1.5.0
   - mne-kit-gui =1.3.0
   - autoreject =0.4.3
+  - antio =0.2.0
```

```suggestion
  - antio =0.3.0
```

I've bumped the conda-forge recipe.